docs/guides/known-issues.md

```diff
@@ -21,3 +21,17 @@ Workaround:
 
 - Use a SystemBinary with [`SYSTEM_BINARY`](../api/config-options.md#SYSTEM_BINARY)
 - Do not use AlpineLinux
+
+## libcurl on slim Docker images
+
+Docker images of the "slim" variety often don't have `libcurl` installed,
+such as [the official Node.js image](https://hub.docker.com/_/node) or [sitespeed.io node](https://hub.docker.com/r/sitespeedio/node).
+If you don't install it manually, then `mongodb-memory-server` will not start up in your container, with the following exception message:
+
+> Instance Exited before being ready and without throwing an error!
+
+This can easily be resolved by installing it in your image:
+
+```sh
+$ apt-get install libcurl4
```

Comment:
```suggestion
apt-get install libcurl4
```

it is good to have `$` to show it is meant for the shell, but in this case it makes copy-pasting unnecessarily more annoying (by including the `$ `)